Demo/Color+Helpers.swift

```diff
@@ -0,0 +1,41 @@
+import UIKit
+
+public extension UIColor {
```

Comment:
I think is fine if you add SweetUIKit to the Demo, the Library though should be dependency-free (if possible).